src/lib/pages/contract-details/index.tsx

```diff
@@ -11,6 +11,8 @@ import { observer } from "mobx-react-lite";
 import { useRouter } from "next/router";
 import { useCallback, useEffect } from "react";
 
+import { DelegationsSection } from "../../components/delegations";
+import { AssetsSection } from "../account-details/components/asset";
```

Comment:
1. move `asset` folder from `account-details/components` to `lib/components`
2. rename the folder to `asset-section`

Reply:
already move in another pr